lib/Client.js

```diff
@@ -1888,9 +1888,14 @@ class Client extends EventEmitter {
     * @arg {Object | Array<Object>} [options.file] [DEPRECATED] A file object (or an Array of them)
     * @arg {Buffer} options.file[].file A buffer containing file data
     * @arg {String} options.file[].name What to name the file
+    * @arg {String} [options.threadID] The ID of the thread channel in the webhook's channel to send the message to
```

Comment:
This might sound better in the context of the function
```suggestion
    * @arg {String} [options.threadID] The ID of the thread channel in the webhook's channel to edit the message in
```